docker/hoodie/hadoop/base/pom.xml

```diff
@@ -17,13 +17,13 @@
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <parent>
-    <artifactId>hoodie-hadoop-docker</artifactId>
-    <groupId>com.uber.hoodie</groupId>
-    <version>0.4.8-SNAPSHOT</version>
+    <artifactId>hudi-hadoop-docker</artifactId>
+    <groupId>org.apache.hudi</groupId>
+    <version>0.1.0-SNAPSHOT</version>
```

Comment:
can we keep it as `0.5.0`?  does the frst release have to be 0.1.0? 
This way there is continuity in versions for users.

Reply:
Done.